exercises/go-counting/canonical-data.json

```diff
@@ -0,0 +1,172 @@
+{
+  "exercise": "go-counting",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Test for black territory on 5x5 board",
+      "property": "territory",
+      "input": {
+        "board": [
+		    "  B  ",
+		    " B B ",
+		    "B W B",
+		    " W W ",
+		    "  W  "
+		],
+        "x": 0,
+        "y": 1
+      },
+      "expected": {
+        "stone": BLACK
+        "territory": [(0, 0), (0, 1), (1, 0)]
```

Comment:
Here is a suggestion I have. What if these were to be `{"x": whatever, "y": whatever}` instead?

For queen-attack that was suggested in https://github.com/exercism/problem-specifications/issues/721 and implemented in https://github.com/exercism/problem-specifications/pull/935
For robot-simulator that was suggested in https://github.com/exercism/problem-specifications/issues/722 and implemented in https://github.com/exercism/problem-specifications/pull/936

Does the same reasoning apply here? Why or why not?